apps/src/music/views/MusicView.jsx

```diff
@@ -71,7 +71,7 @@ class UnconnectedMusicView extends React.Component {
       instructions: null,
       isPlaying: false,
       startPlayingAudioTime: null,
-      currentAudioElapsedTime: 0,
+      currentPlayhead: 0,
```

Comment:
Similar comment here, for me `currentPlayheadPosition` or `currentPlayheadTime` feels a bit clearer